model/patron.py

```diff
@@ -300,6 +278,48 @@ def synchronize_annotations(self, value):
                 _db.delete(annotation)
         self._synchronize_annotations = value
 
+    @property
+    def root_lane(self):
+        """Find the Lane, if any, to be used as the Patron's root lane.
```

Comment:
This code was moved in from circulation. It's a bit tricky because the patron is not directly associated with a Lane; instead, a Lane may be associated with a _list_ of patron types.